crates/linter/src/plugin/strictness/rules/require_strict_types.rs

```diff
@@ -141,7 +143,22 @@ impl Rule for RequireStrictTypesRule {
                 )
                 .with_annotation(Annotation::primary(program.span()))
                 .with_note("The `strict_types` directive enforces strict type checking, which can prevent subtle bugs.")
-                .with_help("Add `declare(strict_types=1);` at the top of your file."),
+                .with_help("Add `declare(strict_types=1);` at the top of your file.")
+                .with_suggestion(program.source, {
+                    let mut plan = FixPlan::new();
+                    let opening_tag = program.statements.iter().find_map(|statement| match statement {
+                        Statement::OpeningTag(tag) => Some(tag),
+                        _ => None,
+                    });
+
+                    plan.insert(
+                        opening_tag.map(|tag| tag.span().end).unwrap_or_else(|| program.span().start).offset,
```

Comment:
I don't know if it's a good solution to fall back to the start of the program. Maybe it's just better not to allow the fix if we don't find an `OpeningTag`?

Reply:
if no opening tag is found, we should insert `<?php declare(strict_types=1); ?>\n` at the start of the file, as that is an html file most likely.